lib/input.js

```diff
@@ -232,9 +232,10 @@ export default class Input {
       const r = buf.slice(33 + off, 65 + off);
       const s = buf.slice(65 + off, 97 + off);
       const v = buf.readUInt8(97 + off);
-      const signer = Input.recoverSignerAddress(sigHashBuf, v, r, s);
-
-      input.setSig(r, s, v, signer);
+      if (v > 0) {
```

Comment:
Do we just want to check `v` here? I assume we should check `r` and `s` to equals 0, too?

Reply:
correct :)